tests/test_client.py

```diff
@@ -0,0 +1,116 @@
+from textwrap import dedent
+from typing import Dict
+
+from git_machete.client import MacheteClient
+from git_machete.git_operations import GitContext, LocalBranchShortName
+from git_machete.annotation import Annotation
+from .mockers import (GitRepositorySandbox, rewrite_definition_file)
+
+
+class TestClient:
```

Comment:
Hmm `TestClient` sounds like some test counterpart of `MacheteClient`, rather than a _test **for** MacheteClient_. I'd rather call it hmmm... `TestReadingDefinitionFile`?

Reply:
@PawelLipski Well, but in the future this test file might consist of more than just a single test that reads definition file, it is supposed to be for the `client.py` file and was named with the same idea as test for `git_operation.py` file, which is named `TestGitOperations`

Reply:
Okay, makes sense, let's leave then